test/upgrades/v2.1.1/Upgrade.t.sol

```diff
@@ -133,7 +133,7 @@ contract UpgradeTest is Test {
 
         (, response) = activeAccount.call(abi.encodeWithSignature("VERSION()"));
         (bytes32 version) = abi.decode(response, (bytes32));
-        assertEq(version, "2.1.1", "wrong version");
+        assert(version != "2.1.0");
```

Comment:
**Thought:** Should we assert here that `assertEq(version, "2.1.2", "wrong version");` to be more specific?

Reply:
prev did this but this breaks backwards compatibility w tests.